server/src/test/java/io/druid/client/DirectDruidClientTest.java

```diff
@@ -314,7 +313,7 @@ public void testQueryInterruptionExceptionLogMessage() throws JsonProcessingExce
 
     TimeBoundaryQuery query = Druids.newTimeBoundaryQueryBuilder().dataSource("test").build();
     HashMap<String, List> context = Maps.newHashMap();
-    interruptionFuture.set(new ByteArrayInputStream("{\"error\":\"testing\"}".getBytes()));
+    interruptionFuture.set(new ByteArrayInputStream("{\"error\":\"testing1\",\"errorMessage\":\"testing2\"}".getBytes()));
```

Comment:
In 0.9.0, 0.9.1 "error" gets swallowed by the broker anyway – nobody would ever see custom error messages unless they were querying historicals/peons directly. In the old code, the custom error was put into the "causeMessage" but that was never written back out to the http response.

This PR makes it so "error" is a specific set of things and "errorMessage" is more free form.